santos_akishagelsey/country/countries.js

```diff
@@ -0,0 +1,44 @@
+async function searchCountry() {
+    const countryInput = document.getElementById('countryInput').value;
```

Comment:
IDs should be written in snake case. All lower case separated by underscore. It should be ('country_input')